external-apis/muoversi2015.e015.servizirl.it.yaml

```diff
@@ -30,18 +30,65 @@ info:
     name: Technical Management Board di E015
     url: http://www.e015.regione.lombardia.it/PE015/esplora-i-contenuti/i-servizi/muoversi-lombardia
     email: e015-tmb@regione.lombardia.it
+    x-publisher: [dati publisher]
+    x-provider: [dati provider servizi]
+    x-complexity-index: [valore della complessità delle api (in FP)]
+  x-gdpr:
+    - data: [Informazione da gestire (ad es. Codice Fiscale)]
+      data-type: [Tipologia (ad es. personale|sensibile|giudiziario)]
+  x-lifecycle:
+    published: [Data-Pubblicazione-Progetto]
+    retires: [Data-Ritiro-Progetto]
+    maturity: [enum di valori stable|deprecated|new|alpha|beta]
+  x-audience: 
+    visibility: [Set di valori che definiscono chi può visualizzare questa API]
+    allowed-profiles: [Elenco dei profili business abilitati a consumare l'API]
+  x-ecosystem:
+    - name: [Nome ecosistema]
+      url: [Url verso ecosistema di cui fa parte questa definizione API]
+      description: [Descrizione Ecosistema]
+  x-catalog:
+    tag: [TAG1,TAG2]
+    category: [CATEGORY1,CATEGORY2]
+    context: 
+      - name: [Nome contesto]
+        description: [Descrizione contesto]
+  x-rating:
+    stats: [URL lettura statistiche KPI API]
+    rating: [URL lettura rating API]
+  x-api-id: [Identificativo univoco insieme di funzionalita']
+  x-business-function:
+    - name: [Nome business function]
+      description: [Descrizione business function]
+  x-technology-type: [Da definire secondo glossario/dominio dei valori]
+  x-glossary: 
```

Comment:
Le info di glossario non le metterei direttamente qui ma in `components.schemas`.

Guarda in questo draft come le informazioni di glossario siano indicate sia sulla classe
che per campo.

https://github.com/teamdigitale/api-openapi-samples/blob/master/external-apis/spid-aa.yaml#L109

Un parser può semplicemente ricostruire tutto il gioco leggendo lo yaml e ricavare l'informazione completa e aggiornata.